RecipeApp/CustomTabBarController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  CustomTabBarController.swift
+//  RecipeApp
+//
+//  Created by Hadley Igoe on 2019-12-19.
+//  Copyright © 2019 Hadley Igoe. All rights reserved.
+//
+
+import UIKit
+
+class CustomTabBarController: UITabBarController {
+    
+    var appCoordinator: ApplicationCoordinator?
+    let recipeBook = RecipeBookCoordinator(navigationController: UINavigationController())
```

Comment:
You're calling this a recipeBook when it's a coordinator....misleading and confusing when you actually use it. Also, why are you handling the two coordinators in different ways?  (one is a var, one is a constant and they're initialized in different places)